subprojects/packagefiles/win-iconv/meson.build

```diff
@@ -0,0 +1,66 @@
+project(
+  'win-iconv',
+  'c',
+  version: '0.0.8',
+  meson_version: '>=1.1.0',
```

Comment:
This seems overly restrictive.  Does this wrap require any features from newer Meson versions?

Reply:
I got a warning, since I didn't specify an explicit version `meson.options` as options files is only supported since 1.1.0 😓

Reply:
Yeah, it's better to call it `meson_options.txt` and let the wrap be used on systems with older Meson.